e2e/test/scenarios/models/reproductions/39150-model-with-same-custom-column-name.cy.spec.js

```diff
@@ -0,0 +1,50 @@
+import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
+import { createQuestion, restore } from "e2e/support/helpers";
+
+const { ORDERS_ID, ORDERS } = SAMPLE_DATABASE;
+
+const modelDetails = {
+  type: "model",
```

Comment:
The bug is there regardless of whether the questions are models or not.
Should we add tests for `type: "question"` as well?

Reply:
I need to double check it, if question is affected then yes